vg/draw/canvas.go

```diff
@@ -294,6 +332,34 @@ func (c Canvas) Crop(minx, miny, maxx, maxy vg.Length) Canvas {
 	}
 }
 
+// Tile returns new Canvases corresponding to the reciever canvas tiled
+// into the given number of rows and columns (row major, starting from the
+// top left corner) where t, b, l, r, h,
+// and w are the padding above, below, to the left, to the right, between rows,
+// and between columns of the tiles, respectively.
+func (c Canvas) Tile(rows, cols int, t, b, l, r, h, w vg.Length) [][]Canvas {
```

Comment:
perhaps define:
```go
type Tiles struct {
 Values []Canvas
 Rows, Cols int
}

func (t Tiles) At(i,j int) Canvas { ... }
```
for more easily ranging through all tiles? (and still retain convenient direct access to each sub-canvas)